sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveTableScanSuite.scala

```diff
@@ -90,4 +90,11 @@ class HiveTableScanSuite extends HiveComparisonTest {
     assert(sql("select CaseSensitiveColName from spark_4959_2").head() === Row("hi"))
     assert(sql("select casesensitivecolname from spark_4959_2").head() === Row("hi"))
   }
+
+  test("SPARK-8003 Resolve spark__partition__id in Hive queries") {
+    TestHive.sql("CREATE TABLE tb (KEY INT, VALUE STRING)")
```

Comment:
This suite is used to compare the result of Spark SQL and Hive. Since `spark__partition__id` is a Spark specific virtual column, the right place for this test is  `org.apache.spark.sql.SQLQuerySuite`. Actually, why do you need this test?